README.md

```diff
@@ -1 +1,12 @@
-# opentelemetry-installer
\ No newline at end of file
+# opentelemetry-installer
+
+This is a simple script to integrate OpenTelemetry in the project. The script `otel_integrator.sh` should be copied to the root of the project.
+First param is a project namespace. Default to `Pyz`.
+Second param is deploy file name. Default to `deploy.yml`
+Third param is install script name. Default to `config/install/docker.yml`
```

Comment:
The third parameter is the install script name, which defaults to config/install/docker.yml.